fastlane/lib/fastlane/actions/opt_out_usage.rb

```diff
@@ -13,9 +13,8 @@ def self.description
       def self.details
         [
           "By default, fastlane will track what actions are being used",
-          "No personal information is shared. More information available on",
-          "https://github.com/fastlane/enhancer",
-          "Add `opt_out_usage` at the top of your Fastfile"
+          "No personal information is shared.",
+          "Add `opt_out_usage` at the top of your Fastfile to disable metrics collection"
```

Comment:
In the README we should mention both I'd say. For the user it's easier to set it once in the Fastfile, instead of setting an env variable on every machine that uses fastlane. Either way works for me 👍